scalajvm/app/models/queries/QuoteQueries.scala

```diff
@@ -73,6 +73,13 @@ case class QuoteQueries(implicit session: DBSession) {
     }.map(rs => Quote(rs)).first().apply()
   }
 
+  def getRandomQuote: Option[Quote] = {
+    val q = Quote.syntax("q")
+    withSQL {
+      select(q.*).from(Quote as q).append(sqls"ORDER BY RANDOM()").limit(1)
```

Comment:
Are you sure we will be satisfied by performance of this query? There are known problems with `order by random`.